packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -103,15 +104,15 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
           setLoadingPeerProfile(false)
 
           // TODO: Better validation
-          if (!response?.data?.peerProfile) {
+
+          if (!('peerProfile' in response.data)) {
```

Comment:
Could you please change that back to `!response?.data?.peerProfile`? This way its more consistent with the rest of the code.